src/config_file.c

```diff
@@ -249,6 +249,17 @@ gboolean load_config(const gchar *filename, RaucConfig **config, GError **error)
 	c->keyring_path = resolve_path(filename,
 			g_key_file_get_string(key_file, "keyring", "path", NULL));
 
+	c->use_cert_signing_time = g_key_file_get_boolean(key_file, "keyring", "use-signing-time", &ierror);
+	if (g_error_matches(ierror, G_KEY_FILE_ERROR, G_KEY_FILE_ERROR_KEY_NOT_FOUND) ||
+			g_error_matches(ierror, G_KEY_FILE_ERROR, G_KEY_FILE_ERROR_GROUP_NOT_FOUND)) {
+		c->use_cert_signing_time = FALSE;
```

Comment:
Why do you have  `use-signing-time` as the config option, but `use_cert_signing_time` as the variable name? The time is not store in the certificate's signature but in the bundle signature. Maybe `use-bundle-signing-time` for both?

Reply:
Sounds good.